agent/consul/fsm/snapshot_oss.go

```diff
@@ -15,7 +15,7 @@ func init() {
 	registerRestorer(structs.KVSRequestType, restoreKV)
 	registerRestorer(structs.TombstoneRequestType, restoreTombstone)
 	registerRestorer(structs.SessionRequestType, restoreSession)
-	registerRestorer(structs.ACLRequestType, restoreACL)
+	registerRestorer(structs.DeprecatedACLRequestType, restoreACL)
```

Comment:
Did you want to also clean up `restoreACL` yet?

Reply:
Don't we still need `restoreACL` for snapshots that contain legacy ACL entries? I kept the test around to show we can still restore snapshots with this data.

Reply:
Ah, right. I wasn't sure how much we were just going to restore old ACL data directly into `/dev/null` or not.